Expensify/expensify-common
# adding new lines rules so it works on mobile

@marcaaron 

New lines were not working on mobile, added a new rule to transform `\n` into <br>

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/129629

# Tests
N/A

# QA
1. Go to a report on the mobile app, add a comment with more than 1 line
1. Verify that it the lines are applied to the comment seen on the report
